### PR TITLE
fix link

### LINKED ISSUE #8642: bad link provided by edeka spider
https://www.openstreetmap.org/way/235612992 has working https://www.edeka.de/eh/minden-hannover/edeka-junghans-poppauer-str.-2/index.jsp

ATP suggests https://www.edeka.de/eh/minden-hannover/edeka-junghans-poppauer-str.-2/index.jspp

(not investigated how it happens and is it a wider issue)

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes #8642

### CODE REVIEW

**Comment by matkoniecz** [comment]
tested by running spider and `cat edeka_de.geojson | grep junghans-poppauer`

**Comment by matkoniecz** [comment]
```
Traceback (most recent call last):
  File "/tmp/opt/python/site-packages/scrapy/utils/defer.py", line 279, in iter_errback
    yield next(it)
          ^^^^^^^^
  File "/tmp/opt/python/site-packages/scrapy/utils/python.py", line 350, in __next__
    return next(self.data)
           ^^^^^^^^^^^^^^^
  File "/tmp/opt/python/site-packages/scrapy/utils/python.py", line 350, in __next__
    return next(self.data)
           ^^^^^^^^^^^^^^^
  File "/tmp/opt/python/site-packages/scrapy/core/spidermw.py", line 106, in process_sync
    for r in iterable:
  File "/tmp/opt/python/site-packages/scrapy/spidermiddlewares/referer.py", line 352, in <genexpr>
    return (self._set_referer(r, response) for r in result or ())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/opt/python/site-packages/scrapy/core/spidermw.py", line 106, in process_sync
    for r in iterable:
  File "/tmp/opt/python/site-packages/scrapy/spidermiddlewares/urllength.py", line 27, in <genexpr>
    return (r for r in result or () if self._filter(r, spider))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/opt/python/site-packages/scrapy/core/spidermw.py", line 106, in process_sync
    for r in iterable:
  File "/tmp/opt/python/site-packages/scrapy/spidermiddlewares/depth.py", line 31, in <genexpr>
    return (r for r in result or () if self._filter(r, response, spider))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/opt/python/site-packages/scrapy/core/spidermw.py", line 106, in process_sync
    for r in iterable:
  File "/tmp/codebuild/output/src1472/src/github.com/alltheplaces/alltheplaces/locations/spiders/edeka_de.py", line 44, in parse
    item["opening_hours"] = self.parse_hours(item)
                            ^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/codebuild/output/src1472/src/github.com/alltheplaces/alltheplaces/locations/spiders/edeka_de.py", line 82, in parse_hours
    for day in store_obj["businessHours"].values():
               ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/tmp/opt/python/site-packages/scrapy/item.py", line 79, in __getitem__
    return self._values[key]
           ~~~~~~~~~~~~^^^^^
KeyError: 'businessHours'
```

File: locations/spiders/edeka_de.py [hunk 1]
```diff
@@ -41,13 +41,7 @@ def parse(self, response):
...
             item = DictParser.parse(store)
 
-            oh = OpeningHours()
-            for day in store["businessHours"].values():
-                if not isinstance(day, dict) or not day["open"]:
-                    continue
-                for rule in day["timeEntries"]:
-                    oh.add_range(day["weekday"], rule["from"], rule["to"])
-            item["opening_hours"] = oh.as_opening_hours()
+            item["opening_hours"] = self.parse_hours(item)
```
Comment (Cj-Malone) [inline locations/spiders/edeka_de.py:44]:
```suggestion
            item["opening_hours"] = self.parse_hours(store)
```

**Comment by iandees** [comment]
Thanks all. Github was having a problem with webhooks and I was waiting for that to get fixed.

**Pull request merged**